docroot/modules/custom/dct_newsletter/dct_newsletter.routing.yml

```diff
@@ -0,0 +1,7 @@
+dct_newsletter.form:
+  path: '/form'
```

Comment:
The newsletter subscription form should be displayed inside a block.

Reply:
I added a route just for testing. I will place the form in a block in another sub-task.